ocaml/xenopsd/scripts/swtpm-wrapper

```diff
@@ -129,24 +170,36 @@ def main(argv):
             if not os.path.exists(urandom):
                 os.mknod(urandom, 0666 | stat.S_IFCHR, os.makedev(1, 9))
 
-            os.chown(tpm_dir, uid, uid)
             if os.path.exists(os.path.join(tpm_dir, ".lock")):
                 os.chown(os.path.join(tpm_dir, ".lock"), uid, uid)
-            os.chown(os.path.join(tpm_dir, STATE_FILE), uid, uid)
+            os.chown(tpm_file, uid, uid)
+
+            if (tpm_state_scheme != "file"):
+                os.chown(tpm_dir, uid, uid)
+            else:
+                os.chown(tpm_dir, 0 , uid)
```

Comment:
Just noting here that this works because xenopsd creates tpm_dir as 0750. After swtpm switches uid/gid, the process gid matches the dir gid and allows swtpm to read files in tpm_dir but not create new ones. It can't change the directory permissions either because that is reserved for the directory owner.

We need to ensure that the xenospd side doesn't ever change to create tpm_dir with more lax permissions at some point in the future.

Reply:
Maybe it should chmod `0750` here as well to ensure that?